js/script.js

```diff
@@ -0,0 +1,18 @@
+//Set the background to the img src to keep alt text on images
+var imgs = document.getElementById("img-container").getElementsByTagName('img');
+document.body.style.backgroundImage = "url(" + imgs[0].src + ")";
+document.body.style.backgroundSize = "100%";
+const numPhotos = imgs.length - 1;
```

Comment:
Same as above -- variable initialized but not being used, could've been used on the nextPhoto function.

Reply:
Oops! That was my original intent, I was suppose to use it in the nextPhoto function. Thank you!